pipeline/projects/sales/catalogs.py

```diff
@@ -217,7 +218,7 @@ def __call__(self, data:dict, non_auctions):
 
 		catalog_label = self.helper.physical_catalog_label(cno, sale_type, owner, copy)
 		row_name = f'STAR Entry for Physical {catalog_label}'
-		row = vocab.EntryTextForm(ident=record_uri, content=content, label=row_name)
+		row = vocab.make_multitype_obj(vocab.EntryTextForm,self.helper.catalog_type(sale_type), ident=record_uri, content=content, label=row_name)
```

Comment:
I'm not sure this one should have typing relating to the sale type. This is an entry in the Getty database of known physical copies of catalogs (an entry in the STAR database). All such entries seem like they should have the same typing, regardless of the type of event the catalog they describe is representing.